data/presets/area/footway.json

```diff
@@ -0,0 +1,22 @@
+{
+    "fields": [
+        "area/highway",
```

Comment:
this field is not necessary anymore in this specific preset, since it would just duplicate the information already present in the title of the preset.

```suggestion
```

Reply:
I admit that I have no idea why exactly it will work but I applied it